uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/token/TokenMvcMockTests.java

```diff
@@ -3071,7 +3044,7 @@ public void testLoginAuthenticationFilter() throws Exception {
                 .param("username", developer.getUserName())
                 .param("user_id", developer.getId())
                 .param(OriginKeys.ORIGIN, developer.getOrigin()))
-                .andExpect(status().isUnauthorized());
+                .andExpect(status().isForbidden());
```

Comment:
I think removal of the non-spec response_type param from the request matcher [here](https://github.com/cloudfoundry/uaa/blob/b227e8302bacc9af510f0a7c27cce579e33dd859/uaa/src/main/webapp/WEB-INF/spring/oauth-endpoints.xml#L166-L178) meant that these requests started getting caught in the [statelessTokenApiSecurity policy](https://github.com/cloudfoundry/uaa/blob/b227e8302bacc9af510f0a7c27cce579e33dd859/uaa/src/main/webapp/WEB-INF/spring/oauth-endpoints.xml#L184-L194) for the first time whereas before they would have matched [loginTokenRequestMatcher (which I think should be removed entirely ASAP)](https://github.com/cloudfoundry/uaa/blob/ab292e78b239e01d4c5dfcccb8c372b6bae1fc29/uaa/src/main/webapp/WEB-INF/spring/login-server-security.xml#L111-L126). Since the tokens in these tests do not have the `uaa.user` scope required by the stateless api security policy they get 403; even if they requested another unexpired token it would not allow them to successfully pass this filter.